ai/docs/Vicuna Model Deployment and Training Documentation.md

```diff
@@ -0,0 +1,53 @@
+# Vicuna Model Deployment and Training Documentation
+## Introduction
+
+[Vicuna model](https://lmsys.org/blog/2023-03-30-vicuna/) is an open-source chatbot model trained by fine-tuning LLaMA on user-shared conversations collected from ShareGPT. It has superior performance compared to LLaMA and Alpaca, even approaching the performance of ChatGPT, evaluted by GPT-4 and with comparably low training costs. The code and model parameters [in open access with github](https://github.com/lm-sys/FastChat).
+
+This document will focus on deploying and fine-tuning Vicuna model. We re-encapsulate the FastChat API for Vicuna, make it easier for personal use.
+
+Before use, you should have **GPU** in your system, with **enough GPU memory****(14GB for Vicuna-7B and 28GB for Vicuna-13B)**. Also, make sure that **docker, docker-compose, jq** are installed in your Linux system. 
+
+## Deployment
```

Comment:
Do we need to install Nvidia-docker before deployment? If so please add more details.